### DISCUSSION TITLE
Travis build configuration matrix

### DISCUSSION THREAD

**Discussion by joshua-white** [GEOS-DEV/GEOS#1523] [Build System]
We currently have 9-10 build targets on Travis, and we should revisit our coverage strategy:

Options:

- Pay more for better Travis VMs
- Lighten the Travis testing, but add in LC Gitlab for extensive coverage (perhaps just prior to merge)
- Trim our current matrix (do we need both gcc8.2 and 8.3, etc)
- Not build LVArray tests when building GEOSX (strong consensus for this one)

**Comment by TotoGaz** [comment]
I think https://github.com/GEOSX/GEOSX/pull/1700 will conclude this discussion?

EDIT: Should we open an issue for the LvArray idea?

**Comment by paveltomin** [comment]
Probably not relevant anymore after https://github.com/GEOS-DEV/GEOS/pull/2571
